backend/staticfiles/admin/css/autocomplete.4a81fc4242d0.css

```diff
@@ -0,0 +1,275 @@
+select.admin-autocomplete {
```

Comment:
Could you delete the `staticfiles` folder?  I believe that we are not using static files because the backend is an API project.